src/components/IndexTable/IndexTable.tsx

```diff
@@ -85,7 +87,6 @@ function IndexTableBase({
     toggle: toggleHasMoreLeftColumns,
   } = useToggle(false);
 
-  const onboardingScrollButtons = useRef(false);
```

Comment:
This ref isn't being used by anything in this file and never gets toggled to `true` 🤔